percival/carrier/configuration.py

```diff
@@ -64,13 +64,13 @@ class IniSectionParameters(object):
     Child classes must implement a self._parameters dictionary of tuples: {<name>: (<value>, <datatype>)}
     """
     def __getattr__(self, name):
-        if name in self._parameters.keys():
+        if name in list(self._parameters.keys()):
```

Comment:
You can just use `in` to check if a key is in a dict
```suggestion
        if name in self._parameters:
```